v22.2/configure-logs.md

```diff
@@ -536,6 +538,54 @@ sinks:
 To ensure that you are protecting sensitive information, also [redact your logs](#redact-logs).
 {{site.data.alerts.end}}
 
+## Log buffering
+
+{% include_cached new-in.html version="v22.2" %} All CockroachDB [log sinks](#configure-log-sinks) support the buffering of log messages, and buffering is enabled by default for the [Fluentd-compatible](#output-to-fluentd-compatible-network-collectors) and [HTTP](#output-to-http-network-collectors) log sinks.Previous to version v22.2, only the [log file](#output-to-files) log sink supported log message buffering.
+
+With log buffering configured, log messages are held in a buffer for a configurable time period or accumulated message size before being written to the target log sink together as a batch. Log buffering helps to ensure consistent low-latency log message writes over the network even in high-traffic, high-contention scenarios.
+
+The following shows a basic log configuration with buffering configured for [log file](#output-to-files) writes:
+
+~~~ yaml
+file-defaults:
+  dir: logs
+  buffering:
```

Comment:
nit: as discussed in the Q&A, `file-defaults` would use `buffered-writes` instead.